README.md

```diff
@@ -26,3 +26,10 @@ Be sure to have all **dependencies** installed !
 Optional:
 - pyenchant
 - zlib
+
+### To install dependencies on Linux:
+- Arch Linux:
+```sudo pacman -S --needed python python-pyqt5 qt5-svg python-lxml python-pyenchant zlib```
+
+- Debian based (Ubuntu, Linux Mint, etc.):
+```sudo apt-get install python3-pyqt5 libqt5svg5 python3-lxml python3-enchant zlibg1```
```

Comment:
On Ubuntu 16.04 I had to add also python3-pyqt5.qtwebkit and use zlib1g  instead of zlibg1 so my depency check would look like this:
```
sudo apt-get install python3-pyqt5 python3-pyqt5.qtwebkit libqt5svg5 python3-lxml python3-enchant zlib1g
```